src/core/account/plugin-api.js

```diff
@@ -69,7 +69,9 @@ export class CurrencyConfig extends Bridgeable<EdgeCurrencyConfig> {
     if (tools.importPrivateKey == null) {
       throw new Error('This wallet does not support importing keys')
     }
-    return tools.importPrivateKey(userInput, opts.keyOptions)
+    const keys = await tools.importPrivateKey(userInput, opts.keyOptions)
+    keys.imported = true
+    return keys
```

Comment:
Can we change this structure to `return { ...keys, imported: true }`. That will keep TypeScript happy after we run the conversion script.